packages/commonwealth/client/scripts/app.ts

```diff
@@ -58,39 +58,43 @@ export async function initAppState(
         app.config.nodes.clear();
         app.user.notifications.clear();
         app.user.notifications.clearSubscriptions();
-        data.nodes
+        data.result.nodes
           .sort((a, b) => a.id - b.id)
           .map((node) => {
             return app.config.nodes.add(NodeInfo.fromJSON(node));
           });
-        data.chains
-          .filter((chain) => chain.active)
+        data.result.chainsWithSnapshots
+          .filter((chain) => chain.chain.active)
           .map((chain) => {
-            delete chain.ChainNode;
+            delete chain.chain.ChainNode;
             return app.config.chains.add(
               ChainInfo.fromJSON({
                 ChainNode: app.config.nodes.getById(chain.chain_node_id),
```

Comment:
I think we missed a change on this line - I found a bug where instead of `chain.chain_node_id` we should have written `chain.chain.chain_node_id`. PR here:  
https://github.com/hicommonwealth/commonwealth/pull/2404

Reply:
Didn't notice this comment and opened a separate PR for this #2407.